lib/discourse_livestream/handle_topic_chat_channel_creation.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module DiscourseLivestream
+  def self.handle_topic_chat_channel_creation(topic)
+    return if topic.category.blank?
+    return if DiscourseLivestream::TopicChatChannel.exists?(topic_id: topic.id)
+    return if topic.tags.blank? || topic.tags.none? { |tag| tag.name == "livestream" }
```

Comment:
This tag may be a good opportunity for a site setting in the future :)

Reply:
Yes, that is on the roadmap :)